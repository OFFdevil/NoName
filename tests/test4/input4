def FUNCa A
{
    // body
}

main
    int A:=123;
  FUNCa( A);