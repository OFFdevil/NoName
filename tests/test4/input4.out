Variable A doesn't exist
